.github/workflows/build-and-test.yml

```diff
@@ -94,20 +105,36 @@ jobs:
            libgmock-dev                                   \
            git                                            \
            libprotobuf-dev
+      - name: Install dependencies (Windows)
+        if: ${{ startsWith(matrix.config.os, 'windows') }}
+        run: |
+          py -3 -m pip install aqtinstall
+          py -3 -m aqt install-qt  --outputdir C:\Qt windows desktop 5.15 win64_msvc2019_64
+          py -3 -m pip install conan==1.54.0
+          choco install -y ccache
+          echo "C:\ProgramData\chocolatey\lib\ccache\tools\ccache-4.7.3-windows-x86_64" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
       - name: Save CCache Timestamp
         id: ccache_timestamp
         run: echo "timestamp=$(date +%m-%d-%Y--%H:%M:%S)" >> $GITHUB_OUTPUT
+        shell: bash
       - name: Setup CCache Files
         uses: actions/cache@v3
         with:
-          path: .ccache
+          path: |
+            .ccache
+            ~/.conan
+            C:\.conan
           key: ${{ matrix.config.build_type }}-${{ matrix.config.cc }}-ccache-${{ steps.ccache_timestamp.outputs.timestamp }}
           restore-keys: |
             ${{ matrix.config.build_type }}-${{ matrix.config.cc }}-ccache-
             ccache-
-      - run: mkdir build
-      - name: CMake Configure
-        working-directory: ./build
+      - run: ccache -p
+      - run: ccache -z
+      - run: mkdir "build_${{ matrix.config.conan_profile }}"
```

Comment:
I don't understand that. `conan_profile` is only set for the MSVC build in the matrix. And even if it were, it might be confusing since we don't use Conan for this build.

Reply:
Removed it here.